src/www/ui/api/documentation/openapi.yaml

```diff
@@ -422,6 +422,47 @@ paths:
                 $ref: '#/components/schemas/Info'
         default:
           $ref: '#/components/responses/defaultResponse'
+    patch:
+      tags:
+      - Upload
+      summary: Update a upload
+      requestBody:
+        description: Information to be updated
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              description: Set the properties to be updated
+              properties:
+                name:
+                  description: New name for the upload
+                  type: string
+                  nullable: true
+                  example:
+                    "Updated upload name"
+                description:
+                  description: New description for the upload
+                  type: string
+                  nullable: true
+                  example:
+                    "Updated upload description"
+      parameters:
+        - name: uploadId
+          description: Upload Id, which upload should be modified
+          in: header
```

Comment:
We should get the upload id from path like `PATCH /api/v2/uploads/123`.
What do you think?

Reply:
And repurpose `PUT` to have one more field like `action=copy|move` like `PUT /folders/{id}` (but having action in in `query` to reduce amount of headers).
https://github.com/Shruti3004/FOSSology-REST-API/blob/d61f7640dc18fce3fb6274c7fa0bfa566ce40045/src/www/ui/api/documentation/openapi.yaml#L961-L990

Reply:
> We should get the upload id from path like `PATCH /api/v2/uploads/123`.
> What do you think?

First I have made ike this but it was moving the upoad since that was called first